sktime/transformations/base.py

```diff
@@ -198,6 +198,29 @@ def __rmul__(self, other):
         else:
             return NotImplemented
 
+    def __or__(self, other):
+        """Magic | method, return MultiplexTranformer.
+
+        Implemented for `other` being either a MultiplexTransformer or a transformer.
+
+        Parameters
+        ----------
+        other: `sktime` transformer or sktime MultiplexTransformer
+
+        Returns
+        -------
+        MultiplexTransformer object
+        """
+        from sktime.transformations.compose import MultiplexTransformer
+
+        if isinstance(other, MultiplexTransformer) or isinstance(
```

Comment:
this expression is the same, logically, as `isinstance(other, BaseTransformer)`, because all `MultiplexTransformer` instances are also `BaseTransformer` instances. `isinstance` checks for subclass instance, not "just" direct instance.

Reply:
that makes sense